src/component/MintNft/MintNftForm/MintNftForm.jsx

```diff
@@ -0,0 +1,214 @@
+import React, { useEffect, useState } from 'react';
+import { ethers } from 'ethers';
+import { useForm, Controller, useFieldArray } from 'react-hook-form';
+import UploadImage from '../../common/Form/UploadImage';
+import Input from '../../common/Form/Input';
+import PrimaryButton from '../../common/Buttons/PrimaryButton';
+import ErrorMessage from '../../common/Form/ErrorMessage';
+import TextArea from '../../common/Form/TextArea';
+import SecondaryButton from '../../common/Buttons/SecondaryButton';
+import { mintNft } from '../../../apis/cryptoApi';
+
+const labels = { name: 'name', description: 'description' };
+
+const options = {
+  name: { required: 'This is required' },
```

Comment:
The validation messages different even though they mean the same, keep it simple like "Please provide ..."